### PR TITLE
Code updated: aware log messages

### PR DESCRIPTION
**Pull request by tinazhang128**

### CODE REVIEW

File: aware-phone/src/main/java/com/aware/phone/ui/Aware_Light_Client.java [hunk 1]
```diff
@@ -469,8 +476,71 @@ protected void onPause() {
...
+
+    @Override
+    protected void onStop() {
+        // Check if the activity is finishing
+        boolean isFinishing = this.isFinishing();
+
+        // Handle based on whether it's system-initiated closure
+        if (!isFinishing) {
+            if (isBatteryOptimizationIgnored(this, "com.aware.phone")) {
+                Log.d("AWARE-Light_Client", "AWARE-Light stopped from background: may caused by battery optimization");
```
Comment (Songyan12345) [inline aware-phone/src/main/java/com/aware/phone/ui/Aware_Light_Client.java:522]: There are three more instances here that could also be changed to "may be caused by", thanks!

**Review by Songyan12345 — CHANGES_REQUESTED** [review]
One more minor comment, thanks!

File: aware-phone/src/main/java/com/aware/phone/ui/Aware_Light_Client.java (hunk 1 above)
Comment (tinazhang128) [reply aware-phone/src/main/java/com/aware/phone/ui/Aware_Light_Client.java]: Changes made!

**Review by Songyan12345 — APPROVED** [review]
LGTM, thanks :)

**Pull request merged**